app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/Evaluation.tsx

```diff
@@ -577,7 +589,10 @@ export default function Evaluation(props: EvaluationProps) {
               borderColor: theme.palette.divider,
             }}
           >
-            <ColorSquare color={KEY_COLOR} />
+            <EvaluationIcon
```

Comment:
option 1 : I think we should keep both the color and eval icon? 🤔 
option 2: Or have EvaluationIcon accept a color and pass KEY_COLOR so the icon has eval color.

The color becomes relevant when comparing evaluation runs and possibly other cases. might be product q

Reply:
can follow up with product and separately address

Reply:
Option 2 sounds good. 
Having both did not look pleasant. In figma, one side used colorsquare, the other side used icon only. It feels off, too.